bindings/rust/s2n-tls-tokio/tests/shutdown.rs

```diff
@@ -281,3 +286,109 @@ async fn shutdown_with_poll_blinding() -> Result<(), Box<dyn std::error::Error>>
 
     Ok(())
 }
+
+#[tokio::test(start_paused = true)]
```

Comment:
Could you add a comment here to explain that it is what lets us "skip" through time? It's a rather opaquely named option 😄

Reply:
I don't think it's reasonable to put a comment on every test with "start_paused", but I expanded the comments about not actually waiting 10min, which are the strange / unexpected parts.